core/contracts/oracle/implementation/Finder.sol

```diff
@@ -5,18 +5,30 @@ import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "../interfaces/FinderInterface.sol";
 
 /**
- * @title Implementation of the FinderInterface.
+ * @title Provides addresses of the live contracts implementing certain interfaces.
+ * @dev Examples are the Oracle or Store interfaces.
```

Comment:
Could be clearer: "Examples of interfaces with implementations that Finder locates are the Oracle and Store interfaces"